packaging/linux/include/mod.in

```diff
@@ -0,0 +1,44 @@
+#!/bin/sh
+
+MINIMUM_MONO_VERSION="4.2"
+
+make_version() {
+  echo "$1" | tr '.' '\n' | head -n 4 | xargs printf "%03d%03d%03d%03d";
+}
+
+mono_missing_or_old() {
+  command -v mono >/dev/null 2>&1 || return 0
+  MONO_VERSION=$(mono --version | head -n1 | cut -d' ' -f5)
+  [ "$(make_version "${MONO_VERSION}")" -lt "$(make_version "${MINIMUM_MONO_VERSION}")" ] && return 0
+  return 1
+}
+
+if mono_missing_or_old; then
+  if command -v zenity > /dev/null
+  then
+    zenity --no-wrap --error --title "{MODNAME}" --text "{MODNAME} requires Mono ${MINIMUM_MONO_VERSION} or greater.\nPlease install Mono using your system package manager."
+  else
+    printf "{MODNAME} requires Mono %s or greater.\nPlease install Mono using your system package manager.\n" "${MINIMUM_MONO_VERSION}"
+  fi
+  exit 1
+fi
+
+HERE="$(dirname "$(readlink -f "${0}")")"
+cd "${HERE}/../lib/openra" || exit 1
+
+export SDL_VIDEO_X11_WMCLASS="openra-{MODID}"
+
+# APPIMAGE is an environment variable set by the runtime
+mono --debug OpenRA.Game.exe Game.Mod={MODID} Engine.LaunchPath="${APPIMAGE}" "$@"
```

Comment:
Hardcoding APPIMAGE here breaks running from an `--appimage-extract`ed filesystem.  I should check if this is empty and fall back to `${0}`.